.github/workflows/test-and-build.yaml

```diff
@@ -0,0 +1,65 @@
+name: Test & Build
+on:
+  push:
+    branches:
+      - master
+      - dev
+      - v*
+      - beta-v*
+      # TODO @lightswitch05 remove this!
+      - feature/introduce-debian-version-tags
+  pull_request:
+
+# TODO @lightswitch05 remove this!
+env:
+  DOCKER_HUB_REPO: lightswitch05
```

Comment:
obviously this and line #10 needs to be removed. I'm leaving it in here until at least an initial review since it makes development on this branch much easier on my repo.

Reply:
removed!